### PR TITLE
Add new command "htmlcss" to allow external stylesheet

### PR DESCRIPTION
**Pull request by rolland-esol**
To allow better integration into other documentation sets, it's useful to be able to configure the stylesheet externally.
This change is expected to be backwards-compatible.

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -36,6 +36,8 @@
 #include "ReqDocumentPdf.h"
 #include "renderingHtml.h"
 
+extern std::string htmlcss;
+
```
Comment (goeb) [inline src/main.cpp:40]: This declaration would better be in req.h, with a comment explaining its purpose (eg: "css file for html output").

File: src/renderingHtml.cpp [hunk 2]
```diff
@@ -22,6 +22,8 @@
 #include "global.h"
 #include "stringTools.h"
 
+std::string htmlcss;
+
```
Comment (goeb) [inline src/renderingHtml.cpp:26]: This instantiation would better be in req.cpp with the other "static objects".

File: test/stylesheet.css [hunk 3]
```diff
@@ -0,0 +1,22 @@
+body { font-family: Verdana,sans-serif; }
```
Comment (goeb) [inline test/stylesheet.css:1]: This file is not used. Have you forgotten to commit a related test script?
If no related test, could you please move it to the "docs" directory

**Review by goeb — CHANGES_REQUESTED** [review]
I will be glad to take your contribution.
Please have a look at my remarks.

File: test/stylesheet.css (hunk 3 above)
Comment (rolland-esol) [reply test/stylesheet.css:1]: Will move to docs/.

File: src/main.cpp (hunk 1 above)
Comment (rolland-esol) [reply src/main.cpp:40]: OK

File: src/renderingHtml.cpp (hunk 2 above)
Comment (rolland-esol) [reply src/renderingHtml.cpp:26]: OK

**Pull request merged**